torch/csrc/jit/frontend/schema_matching.cpp

```diff
@@ -327,88 +407,90 @@ static c10::optional<MatchedSchema> tryMatchSchema(
-  for (size_t schema_i = 0; schema_i < schema.arguments().size(); ++schema_i) {
-    const auto& arg = schema.arguments()[schema_i];
-    c10::optional<NamedValue> actual_named_value;
-    if (arg.name() == "self" && self) {
-      actual_named_value = self;
-      self = c10::nullopt;
-    } else if (!arg.kwarg_only() && used_args < args.size()) {
-      // Try to convert all the remaining non-kwarg arguments (used_args) to a
-      // list. Allow zeros(IntArrayRef sizes) to work with zeros(1, 2) or
-      // zeros(1)
-      if (allow_conversions && varargsCanBeUsedAsList(schema, schema_i, arg)) {
-        auto value = args[used_args].value(graph);
-        const auto& actual_type = value->type();
-        // The actual cannot already be a list
-        if (actual_type->kind() != TypeKind::ListType &&
-            !convertibleToList(actual_type, unwrapOptional(arg.type()))) {
-          auto formal_type = unwrapOptional(arg.type())
-                                 ->expectRef<ListType>()
-                                 .getElementType();
-
-          Value* list = tryCreateList(
-              formal_type,
-              graph,
-              loc,
-              at::ArrayRef<NamedValue>(args).slice(used_args),
-              failure_messages,
-              err,
-              allow_conversions,
-              type_env);
-          if (!list) {
-            return c10::nullopt;
+    for (size_t schema_i = 0; schema_i < schema.arguments().size(); ++schema_i) {
+      const auto& arg = schema.arguments()[schema_i];
+      c10::optional<NamedValue> actual_named_value;
+      if (arg.name() == "self" && self) {
+        actual_named_value = self;
+        self = c10::nullopt;
+      } else if (!arg.kwarg_only() && used_args < args.size()) {
+        // Try to convert all the remaining non-kwarg arguments (used_args) to a
+        // list. Allow zeros(IntArrayRef sizes) to work with zeros(1, 2) or
+        // zeros(1)
+        if (allow_conversions && varargsCanBeUsedAsList(schema, schema_i, arg)) {
+          auto value = args[used_args].value(graph);
+          const auto& actual_type = value->type();
+          // The actual cannot already be a list
+          if (actual_type->kind() != TypeKind::ListType &&
+              !convertibleToList(actual_type, unwrapOptional(arg.type()))) {
+            auto formal_type = unwrapOptional(arg.type())
+                                  ->expectRef<ListType>()
+                                  .getElementType();
+
+            Value* list = tryCreateList(
+                formal_type,
+                graph,
+                loc,
+                at::ArrayRef<NamedValue>(args).slice(used_args),
+                failure_messages,
+                err,
+                allow_conversions,
+                type_env);
+            if (!list) {
+              return c10::nullopt;
+            }
+            used_args = args.size();
+            positional_inputs.push_back(list);
+            continue;
```

Comment:
This doesn't look like the change relating to this PR. It. looks like a clang-tidy clean up. Can you address this in another PR? It's a little confusing.

Reply:
Thanks, good catch! Remind me to check what `clang_format_all.py` is doing next time... 🙄